lstchain/scripts/lstchain_tune_nsb_waveform.py

```diff
@@ -82,13 +82,13 @@ def main():
 
     config = read_configuration_file(args.config)
 
-    nsb_correction_ratio, data_ped_variance, mc_ped_variance = \
+    extra_nsb_rate, data_ped_variance, mc_ped_variance = \
         calculate_required_additional_nsb(args.input_mc, args.input_data,
                                           config=Config(config))
 
     dict_nsb = {
         "nsb_tuning": True,
-        "nsb_tuning_ratio": np.round(nsb_correction_ratio, decimals=2),
+        "nsb_tuning_rate": np.round(extra_nsb_rate, decimals=3),
```

Comment:
didn't you rename to `nsb_tuning_rate_GHz`?

Reply:
Yes, thanks!  I had just realized... 
Don't waste time with it yet, still WIP...